orc8r/tools/fab/dev_utils.py

```diff
@@ -192,6 +198,70 @@ def get_hardware_id_from_vagrant(vm_name: str) -> str:
     return str(hardware_id)
 
 
+def get_hardware_id_from_docker(
```

Comment:
Which hwid is this grabbing? We already have a fab cmd somewhere for grabbing the hwid from an agw vm

Reply:
This gets the hwid from magmad that runs in docker (this is for feg)